api/tests/routes/backoffice_v3/finance_test.py

```diff
@@ -115,6 +115,58 @@ def test_cancel_already_validated_incident(self, authenticated_client):
         assert history_models.ActionHistory.query.count() == 0
 
 
+class ValidateIncidentTest(PostEndpointHelper):
+    endpoint = "backoffice_v3_web.finance_incident.validate_finance_incident"
+    endpoint_kwargs = {"finance_incident_id": 1}
+    needed_permission = perm_models.Permissions.MANAGE_INCIDENTS
+
+    def test_validate_incident(self, authenticated_client):
+        booking_incident = finance_factories.IndividualBookingFinanceIncidentFactory(
+            booking__pricings=[finance_factories.PricingFactory(status=finance_models.PricingStatus.INVOICED)]
+        )
+
+        response = self.post_to_endpoint(authenticated_client, finance_incident_id=booking_incident.incident.id)
+
+        assert response.status_code == 200
+
+        content = html_parser.content_as_text(response.data)
+        assert "L'incident a été validé avec succès." in content
+
+        updated_incident = finance_models.FinanceIncident.query.get(booking_incident.incidentId)
+        assert updated_incident.status == finance_models.IncidentStatus.VALIDATED
+
+        venue_action = history_models.ActionHistory.query.filter(
+            history_models.ActionHistory.venue == updated_incident.venue
+        ).first()
+        beneficiary_action = history_models.ActionHistory.query.filter(
+            history_models.ActionHistory.user == booking_incident.beneficiary
+        ).first()
+        incident_action = history_models.ActionHistory.query.filter(
+            history_models.ActionHistory.financeIncident == updated_incident
+        ).first()
+
+        assert venue_action and venue_action.actionType == history_models.ActionType.FINANCE_INCIDENT_VENUE_ALERT
+        assert incident_action and incident_action.actionType == history_models.ActionType.FINANCE_INCIDENT_VALIDATED
+        assert (
+            beneficiary_action
+            and beneficiary_action.actionType == history_models.ActionType.FINANCE_INCIDENT_USER_RECREDIT
+        )
+
+    @pytest.mark.parametrize(
+        "initial_status", [finance_models.IncidentStatus.CANCELLED, finance_models.IncidentStatus.VALIDATED]
+    )
+    def test_not_validate_incident(self, authenticated_client, initial_status):
+        finance_incident = finance_factories.FinanceIncidentFactory(status=initial_status)
+
+        response = self.post_to_endpoint(authenticated_client, finance_incident_id=finance_incident.id)
+
+        assert response.status_code == 200
```

Comment:
200 en cas d'échec ? Par homogénéité du backoffice, en général après un POST on fait une redirection vers la page de l'incident.